defaults/main.yml

```
@@ -4,3 +4,5 @@ rhc_auth: {}
 rhc_organization: null
 rhc_server: {}
 rhc_state: "present"
+rhc_insights: 
+  state: "present"
```

```suggestion
  state: present
```
The Ansible style is not to use quotes unless you need them.